app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -79,7 +79,7 @@ CONFIG_INTEL_ADSP_IPC=y
 CONFIG_WATCHDOG=y
 CONFIG_LL_WATCHDOG=y
 
-CONFIG_MEMORY_WIN_2_SIZE=12288
+CONFIG_MEMORY_WIN_2_SIZE=16384
```

Comment:
I wonder how we can make this scale better. Did @marcinszkudlinski you have ideas on this, I think you hit this with tracing backend as well.

I now pushed first PR for shell support https://github.com/zephyrproject-rtos/zephyr/pull/72738 . I was contemplating on adding it to a new slot (and increase WIN_2 size), but:
- shell won't be enable in all builds (just like tracing and probably this telemetry2 as well)
- having to touch the CONFIG_MEMORY_WIN_2_SIZE requires touching multiple board specific files, so enabling the feature with a simpler overlay no longer works

No better solution yet. For shell, I now reuse the dw-slot for mtrace and force exclusivity between shell and mtrace.